ci/scripts/audit.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -eux
+
+pushd the-train
+    mvn ossindex:audit
```

Comment:
Should this be calling `make audit`? This is how it was done on babbage although there it was to make it easier to do both java and js auditing from the same step so might not be a consideration here. I notice The-Train doesn't currently have a make file although I recall @janderson2  suggesting it should possibly have one. The Trello task references a makefile so perhaps it should?